app/javascript/layouts/themes/pupilfirst-dark.css

```diff
@@ -0,0 +1,47 @@
+.theme-pupilfirst-dark {
```

Comment:
@mpraglowski This would be a good idea to customize the brand color palette. I'm not sure about the dark-mode implementation here.
TailwindCSS V2 supports toggling dark mode manually.  [https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually](url)

Reply:
That's just a sample usage od themes. Not my goal for now. We could just drop it for now.